yarn.lock

```diff
@@ -14,6 +14,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@assemblyscript/loader@npm:^0.9.4":
```

Comment:
I'm surprised at the size of the IPFS dependency's footprint

Reply:
Blame on `ipfs-only-hash: ^4.0.0` 😅